app/controllers/newsletter_signups_controller.rb

```diff
@@ -1,17 +1,13 @@
 class NewsletterSignupsController < ApplicationController
 
-  NewsletterSignup = Struct.new(:first_name, :last_name, :email, keyword_init: true)
+  NewsletterSignup = Struct.new(:newsletter_signup, keyword_init: true)
```

Comment:
All of this is really confusing. Why do we need a string? Why can't the model just do the thing?